### ISSUE TITLE
Template linting in a Workspace with multiple folders

### ISSUE THREAD

**Issue by danDanV1** [emberwatch/vscode-ember#10]
When using VSCode workspace with multiple folders/apps, template linting doesn't run, however the .js files are linted properly.
eg.
```
My Workspace
     frontend-ember-app
     backend-ember-app
```
     
I have made a reproduction of the issue with step-by-step instructions in the README.md here:
https://github.com/edeis53/ember-vscode-templatelint-issue/blob/master/README.md

I originally thought it was an issue with Ember-template-lint and made an issue that documents it a bit more here: https://github.com/ember-template-lint/ember-template-lint/issues/813

TLDR; template linting of a workspace with multiple folders works as intended if I install the `lifeart.vscode-ember-unstable` extension instead. 

Perhaps this commit will cover it? https://github.com/lifeart/vscode-ember/commit/3a2598dd64c0ad783f8be6bff50bf4b16609d0e7
